src/quizzes/chapter-2/2.5/security-quiz.js

```diff
@@ -0,0 +1,28 @@
+const rawQuizdown= `
+
+---
+shuffleQuestions: true
+---
+
+# In case you're worried about security risks stemming from dependency versioning, which of the security tools mentioned above would you use if you wanted to ensure your dependencies get safely updated?
```

Comment:
```suggestion
# Which tool is most effective for automatically upgrading libraries to reduce security vulnerabilities?
```

Want to avoid the word 'dependencies' in hopes to make this a little more challenging